test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/testproviders/PersistenceTestProvider.java

```diff
@@ -85,6 +90,134 @@ public void testControllerPersistence(WorkbenchKieServerScenario deploymentScena
         }
     }
 
+
+    // Verifies https://issues.redhat.com/browse/RHPAM-2762
+    public void testAdminUserPasswordChange(WorkbenchKieServerScenario deploymentScenario) {
+        final String NEW_PASSWORD = "newpassword1234!";
+        final String oldPassword = deploymentScenario.getWorkbenchDeployment().getPassword();
+
+        final String username = deploymentScenario.getWorkbenchDeployment().getUsername();
+        String containerId = "testAdminUserPasswordChange";
+        final String workbenchUrl = deploymentScenario.getWorkbenchDeployment().getUrl().toString();
+        final String kieServerUrl = deploymentScenario.getKieServerDeployment().getUrl().toString();
+
+        KieServerControllerClient kieControllerClient = KieServerControllerClientProvider.getKieServerControllerClient(deploymentScenario.getWorkbenchDeployment());
+        KieServicesClient kieServerClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerDeployment());
+
+        KieServerInfo serverInfo = kieServerClient.getServerInfo().getResult();
+        try {
+            WorkbenchUtils.saveContainerSpec(kieControllerClient, serverInfo.getServerId(), serverInfo.getName(), containerId, containerId, Kjar.DEFINITION, KieContainerStatus.STARTED);
+            // Check if old user can connect
+            SoftAssertions.assertSoftly(softly -> {
+                softly.assertAlso(isUserAuthorizedInWorkbench(workbenchUrl, username, oldPassword));
+                softly.assertAlso(isUserAuthorizedInKieServer(kieServerUrl, username, oldPassword));
+            });
```

Comment:
I don't think we need to assert old users can connect as if not, it would have failed before (to kie server when connecting to business central or when deploying the container).